dotnetv3/dynamodb/README.md

```diff
@@ -16,49 +16,31 @@ Amazon DynamoDB is a fully managed, serverless, key-value NoSQL database designe
 
 ### Single actions
```

Comment:
I can't add a comment for lines that didn't change, but I saw that the Overview says "request, import, and manage certificates", but this is the DynamoDB example...

Reply:
Ooops. Fixed.